specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -8318,6 +8952,253 @@
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CapacityReservationGroup"
+          },
+          "description": "The list of capacity reservation groups"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of capacity reservation Groups. Call ListNext() with this URI to fetch the next page of capacity reservation Groups."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The List capacity reservation Group with resource group response."
+    },
+    "CapacityReservationInstanceView": {
+      "type": "object",
+      "properties": {
+        "utilizationInfo": {
+          "$ref": "#/definitions/CapacityReservationUtilization",
+          "description": "Unutilized capacity of the capacity reservation."
+        },
+        "statuses": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InstanceViewStatus"
+          },
+          "description": "The resource status information."
+        }
+      },
+      "description": "The instance view of a capacity reservation."
+    },
+    "CapacityReservationUtilization": {
+      "type": "object",
+      "properties": {
+        "virtualMachinesAllocated": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResourceReadOnly"
+          },
+          "readOnly": true,
+          "description": "A list of references to all virtual machines allocated in the Capacity reservation."
+        }
+      },
+      "description": "capacity reservation utilized capacity."
+    },
+    "CapacityReservationInstanceViewWithName": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the capacity reservation."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/CapacityReservationInstanceView"
+        }
+      ],
+      "description": "The instance view of a capacity reservation that includes the name of the capacity reservation. It is used for the response to the instance view of a capacity reservation group."
+    },
+    "CapacityReservationProperties": {
+      "type": "object",
+      "properties": {
+        "reservationId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique id generated and assigned to the capacity reservation by the platform. <br><br> Does not change throughout the lifetime of the capacity reservation."
+        },
+        "virtualMachinesAssociated": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResourceReadOnly"
+          },
+          "readOnly": true,
+          "description": "A list of references to all virtual machines associated in the capacity reservation."
```

Comment:
A list of references to all virtual machines associated with the capacity reservation.

Reply:
updated